csun-metalab/nomi
# NAM-354 -- UI: Organize the SCSS and conform to BEM

# Description
Refactored CSS to BEM standards; refactored login page from 2 modules to 1 single responsive module.
  
# Testing
1. Ensure all functionality within login page works.
2. Test login page responsiveness and feature set.
3. Check all routes/pages to see if all CSS attributes are applied.
4. Ensure that BEM is applied to all CSS tags within our new CSS file structure.
5. Ensure login input scrolls when the users keyboard covers the screen.
